test/language_server/project/node_test.rb

```diff
@@ -0,0 +1,33 @@
+require 'test_helper'
+
+module LanguageServer
+  class Project
+    class NodeTest < Minitest::Test
```

Comment:
Please don't add test in this PR.
This PR should contain only guard related things.